### PR TITLE
Add absolute links

### PR DESCRIPTION
**Pull request by doruradu**
**What problem is this solving?**

We have a client that doesn't want to integrate the blog into vtex. So we needed a prop "absoluteLinks" that if set to true, will redirect the user to the external blog when clicking a post from any wordpress-integration blocks.

**How should this be manually tested?**

Link the wordpress-integration and add a prop "absoluteLinks:true" to any wordpress-integration block. Example:

`"blog-latest-posts-preview.wordpress-latest-posts-preview":{
    "props":{
      "numberOfPosts":4,
      "showCategories":false,
      "showExcerpts":true,
      **"absoluteLinks":true**
    }
  },`

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "wordpress-integration",
   "vendor": "vtex",
-  "version": "2.0.0",
+  "version": "2.0.1",
```
Comment (polishq) [inline manifest.json:4]:
```suggestion
  "version": "2.0.0",
```

Comment (polishq) [reply manifest.json:4]: No need to increment the version number. When the PR is merged, a GitHub bot will take care of this automatically.

Comment (polishq) [reply manifest.json:4]: Actually, leave it as 2.0.1, since we're publishing that version shortly in a separate PR.

**Comment by doruradu** [comment]
Hi, @polishq! Sorry for the late response, I didn't get any notification from git 🤦‍♂️ 
I've updated the docs, hope the description is ok.

Thanks!

**Pull request merged**